docs/source/release_notes.rst

```diff
@@ -8,6 +8,7 @@ Release Notes
         * Added support for ``scikit-learn`` ``v0.24.0`` :pr:`1733`
     * Fixes
         * Removed ``random_state`` arg from ``get_pipelines`` in ``AutoMLSearch`` :pr:`1719`
+        * Pinned pyzmq at less than 22.0.0 till we add support :pr:`1756`
```

Comment:
lol I don't really know if this is our issue 🤷

Reply:
@angela97lin what would you say here?

Reply:
@dsherry Hmmm, maybe something along the lines of "Pinned pyzmq < 22.0.0 until nbval supports pyzmq == 22.0.0"? It looks like that's where the attempt to install is happening and failing...